### PR TITLE
Add features catalogue and manual checklist

### LINKED ISSUE #67: Use django-debreach
https://github.com/lpomfrey/django-debreach

### LINKED ISSUE #115: Manual checklist when creating project
Some things break too easily and are hard to be spotted ie: hot reloader and auto-prefixer.

What do you guys think of having something like this:
screen shot 2017-06-28 at 04 50 44

### LINKED ISSUE #122: Create a catalog for the project
Currently while following the setup instructions the project includes 171MB of node,js utilities.  Combined with the 117MB brought in through the python dependencies it would be a good exercise to catalog what is being included and why.

### LINKED ISSUE #171: Change celery beat scheduler to avoid using filesystem
The default celery beat scheduler uses a file database to manage the periodic tasks. This is not safe in heroku file system so we should use an alternative.

Some alternatives are:
* https://github.com/sibson/redbeat
* https://github.com/celery/django-celery-beat

### PR DESCRIPTION
**Pull request by rvlb**
## Description

Closes #67 (django-debreach)
Closes #115 (basic checklist)
Closes #122 (catalogue)
Closes #171 (celery redbeat)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires dependencies updates.
- [ ] I have updated the dependencies accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,45 @@ Some settings defaults were decided based on Vinta's experiences. Here's the rat
...
+### Backend
+- `django` for building backend logic using Python
+- `djangorestframework` for building a REST API on top of Django
+- `django-webpack-loader` for rendering the bundled frontend assets
+- `django-js-reverse` for easy handling of Django URLs on JS
+- `psycopg2` for using PostgreSQL database
+- `sentry-sdk` for error monitoring
+- `python-decouple` for easy working with environment variables
+- `celery` for background worker tasks
+- `django-debreach` for performing safer requests
```
Comment (fjsj) [inline README.md:222]: for additional protection against BREACH attack

File: README.md [hunk 2]
```diff
@@ -183,6 +183,45 @@ Some settings defaults were decided based on Vinta's experiences. Here's the rat
...
+  - `react-hot-loader` for improving QoL while developing through automatic browser refreshing
+
+### Backend
+- `django` for building backend logic using Python
+- `djangorestframework` for building a REST API on top of Django
+- `django-webpack-loader` for rendering the bundled frontend assets
+- `django-js-reverse` for easy handling of Django URLs on JS
+- `psycopg2` for using PostgreSQL database
+- `sentry-sdk` for error monitoring
+- `python-decouple` for easy working with environment variables
```
Comment (fjsj) [inline README.md:220]: for reading environment variables on settings files

File: backend/project_name/settings/base.py [hunk 3]
```diff
@@ -127,6 +128,9 @@ def base_dir_join(*args):
 CELERY_ACKS_LATE = True
 CELERY_TIMEZONE = TIME_ZONE
 
+# Redbeat
+redbeat_redis_url = config("REDBEAT_REDIS_URL", default="")
```
Comment (fjsj) [inline backend/project_name/settings/base.py:132]: does this affect localhost anyway (dev running)?
does this need to be lowercase?

**Review by fjsj — CHANGES_REQUESTED** [review]
LGTM overall, but made some comments

File: backend/project_name/settings/base.py (hunk 3 above)
Comment (rvlb) [reply backend/project_name/settings/base.py:132]: It doesn't affect, I think I can move it into the production file.

About the lowercase, yes https://redbeat.readthedocs.io/en/latest/config.html#celery-3-x-config-names (we're using > 4)

**Review by fjsj — APPROVED** [review]
LGTM

**Pull request merged**